packages/abstract-app/src/msgs.rs

```diff
@@ -0,0 +1,16 @@
+#[macro_export]
+/// Groups code that is needed on every app.
+/// This registers the types for safety when using Messages
+/// This is also used to indicate that The Query And Execute messages or used as app messages
+macro_rules! app_messages {
```

Comment:
Can you apply this macro in the example in `abstract-app/examples/counter.rs`?

Reply:
https://github.com/AbstractSDK/contracts/pull/371/commits/d919315ae6b9ce0eaccaf378f552d33421ff9b83